scripts/grunt.sh

```diff
@@ -24,15 +24,23 @@ case "$1" in
     PUBLISHED_PORT=9000
     LIVERELOAD_PORT=35731
     VOLUME_ROOT="/opt/schema_editor"
-    IMAGE_NAME="quay.io/azavea/driver-editor:latest"
+    if [[ -z "${DOCKER_REPOSITORY}" ]]; then
+      IMAGE_NAME="quay.io/azavea/driver-editor:latest"
```

Comment:
This script can only get run in development, so I think this can just be `driver-editor:latest` and remove the check for `DOCKER_REPOSITORY` (in both cases here).